lib/nuntiux/mocker.ex

```diff
@@ -102,25 +156,39 @@ defmodule Nuntiux.Mocker do
     reregister(process_name, mocker_pid, process_pid)
     :proc_lib.init_ack({:ok, mocker_pid})
 
-    loop(%{
-      process_name: process_name,
-      process_pid: process_pid,
-      process_monitor: process_monitor,
-      history: [],
-      opts: opts
-    })
+    opts = Keyword.merge(@default_opts, opts)
+
+    state =
+      Map.merge(@default_state, %{
+        process_name: process_name,
+        process_pid: process_pid,
+        process_monitor: process_monitor,
+        opts: opts
+      })
+
+    loop(state)
   end
 
   @spec call(process_name, request) :: ok
         when process_name: Nuntiux.process_name(),
-             request: request(),
-             ok: history() | received?()
+             request: request_call(),
+             ok: result_call()
   defp call(process_name, request) do
     label = :"$nuntiux.call"
     {:ok, result} = :gen.call(process_name, label, request)
     result
   end
 
+  @spec cast(process_name, request) :: ok
+        when process_name: Nuntiux.process_name(),
+             request: request_cast(),
+             ok: :ok
+  defp cast(process_name, request) do
+    label = :"$nuntiux.cast"
+    send(process_name, {label, request})
+    :ok
+  end
```

Comment:
`cast` alongside `call`. I rather like how the code developed into this naturally.